src/main/frontend/fs/sync.cljs

```diff
@@ -264,7 +269,10 @@
 (defn- get-json-body [body]
   (or (and (not (string? body)) body)
       (or (string/blank? body) nil)
-      (js->clj (js/JSON.parse body) :keywordize-keys true)))
+      (try (js->clj (js/JSON.parse body) :keywordize-keys true)
```

Comment:
Catch json-parse exception, log the body content in the console, and return the exception